### PR TITLE
Nodejs plugin AWSDynamoDB, AWSSNS,  AWSSQS component IDs

### PR DESCRIPTION
**Pull request by tom-pytel**
For: https://github.com/apache/skywalking-nodejs/pull/100

Also includes some previous component server mappings which I guess I forgot to push.

### CODE REVIEW

**Review by wu-sheng — APPROVED** [review]
We don't need extra logos for those services, right?

File: oap-server/server-starter/src/main/resources/component-libraries.yml [hunk 1]
```diff
@@ -425,6 +425,15 @@ tls:
...
   languages: Java
+AWSDynamoDB:
+  id: 132
+  languages: Java,C#,Node.js,Python
+AWSSNS:
+  id: 133
+  languages: Java,C#,Node.js,Python
+AWSSQS:
+  id: 134
+  languages: Java,C#,Node.js,Python
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/component-libraries.yml:436]: 132-134 have been registered. Please register to new IDs.

**Review by wu-sheng — CHANGES_REQUESTED** [review]

**Comment by tom-pytel** [comment]
> We don't need extra logos for those services, right?

They would be logos from Amazon itself, I am guessing copyrighted?

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
One more, changes should be uodated.

**Comment by tom-pytel** [comment]
> One more, changes should be uodated.

Yes, will update in a bit.

File: oap-server/server-starter/src/main/resources/component-libraries.yml (hunk 1 above)
Comment (tom-pytel) [reply oap-server/server-starter/src/main/resources/component-libraries.yml]: Fixed.

**Comment by wu-sheng** [comment]
Merged first, in case of conflicts.

**Pull request merged**